cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1091,42 +1121,22 @@ def make_resource(self, data, **kwargs):
         return CustomAction(**data)
 
 
-class HeadNodeCustomActionsSchema(BaseSchema):
+class QueueCustomActionsSchema(BaseSchema):
     """Represent the schema for all available custom actions."""
 
-    on_node_start = fields.Nested(HeadNodeCustomActionSchema, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
-    on_node_configured = fields.Nested(HeadNodeCustomActionSchema, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
-    on_node_updated = fields.Nested(
-        HeadNodeUpdatableCustomActionSchema, metadata={"update_policy": UpdatePolicy.SUPPORTED}
-    )
+    on_node_start = OneOrManyCustomActionField(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
```

Comment:
question, shouldn't this be `UpdatePolicy.QUEUE_UPDATE_STRATEGY`? (also for the field below)

Reply:
yep, good catch, I'm not very good at cut and past, let's be DRYer on this one